code/datums/components/riding/riding_mob.dm

```diff
@@ -216,6 +218,14 @@
 	else if(ride_check_flags & CARRIER_NEEDS_ARM) // fireman
 		human_parent.buckle_lying = 90
 
+/datum/component/riding/creature/post_vehicle_mob_buckle(mob/living/ridden, mob/living/rider)
+	if(!require_minigame|| ridden.faction.Find(REF(rider)))
+		return
+	ridden.Shake(duration = 2 SECONDS)
+	ridden.balloon_alert(rider, "tries to shake you off.")
```

Comment:
```suggestion
	ridden.balloon_alert(rider, "tries to shake you off!")
```

i think `!` is fine here because it's an alert to a potentialy unfavorable state, we don't want `.` in balloon alerts anyways